src/code/code_800C3C20.c

```diff
@@ -5,9 +5,9 @@ u8 sSfxBankIds[] = {
 };
 
 void AudioMgr_StopAllSfx(void) {
-    u32 i;
+    volatile u8* bank;
```

Comment:
Note, I added volatile because it was needed for the similar function in MM. It is NOT needed here, so if it is preferred I can remove it.